tools/ea/Ea.hs

```diff
@@ -93,7 +93,7 @@ defCoinContract :: FilePath
 defCoinContract = "pact/coin-contract/load-coin-contract.yaml"
 
 defGrants :: FilePath
-defGrants = "pact/genesis/testnet00/grants.yaml"
+defGrants = "pact/genesis/testnet02/grants.yaml"
```

Comment:
Oversight.

Reply:
Again, I'd probably replace this with just `testnet` if we're only ever going to have one floating around at a time.